### DISCUSSION TITLE
Security overview - missing branch protection info

### DISCUSSION THREAD

**Discussion by matthieuguilpin** [community/community#40929] [Code Security]
Hello,

Is it planned to have the branch protection info (x peer reviews required to merge) for each repo in the security section (orga level)?

Best,
Matthieu

**Comment by ossanna16** [comment]
Hi there @matthieuguilpin and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
